src/pages/api/v0/wallet/dex-stab-fees.ts

```diff
@@ -0,0 +1,71 @@
+import BigNumber from "bignumber.js";
+import { NextApiRequest, NextApiResponse } from "next";
+import { EnvironmentNetwork } from "@waveshq/walletkit-core";
+import Cors from "cors";
+import { runMiddleware } from "../../../../utils/middleware";
+import {
+  getAllPoolpairs,
+  getPoolpairsWithStabilizationFee,
+  PoolPairData,
+} from "./poolpairs.utils";
+
+type Fees = {
+  fee: string | BigNumber;
+};
+
+type Error = {
+  error: string;
+};
+
+type DexStabFeeRequestBody = {
+  tokenADisplaySymbol: string;
+  tokenBDisplaySymbol: string;
+  network: EnvironmentNetwork;
+};
+
+interface DexStabRequest extends NextApiRequest {
+  query: DexStabFeeRequestBody;
+}
+
+export const cors = Cors({
+  methods: ["GET", "HEAD"],
+});
+
+export default async function handler(
+  req: DexStabRequest,
+  res: NextApiResponse<Fees | Error>
+) {
+  await runMiddleware(req, res, cors);
+  const { tokenADisplaySymbol, tokenBDisplaySymbol, network } = req.query;
+
+  if (!(tokenADisplaySymbol && tokenBDisplaySymbol)) {
+    return res.status(400).send({
+      error: "Required fields: tokenADisplaySymbol, tokenBDisplaySymbol",
+    });
+  }
+
+  let fee: string | BigNumber = "0";
```

Comment:
do we need `BigNumber`?
```suggestion
  let fee: string = "0";
```